hc/settings.py

```diff
@@ -15,11 +15,11 @@
 
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
-HOST = "localhost"
+HOST = "healthchecks-vulcan.herokuapp.com"
```

Comment:
Host needs to be derived from the environment. Hard-coding this might yield unexpected results
We need to make our setting sensitive to various environments